mon-aide-cyber-api/test/api/demandes/routeAPIDemandeDevenirAidant.spec.ts

```diff
@@ -354,20 +355,18 @@ describe('Le serveur MAC, sur  les routes de demande pour devenir Aidant', () =>
           expect(fauxServiceDeChiffrement.aEteAppele()).toBe(true);
         });
 
-        it('Vérifie que le mail est bien rattaché à la demande', async () => {
-          const demande = unConstructeurDeDemandeDevenirAidant()
-            .avecUnMail('jean.dupont@mail.fr')
-            .construis();
+        it('Vérifie si la demande est déjà traitée', async () => {
```

Comment:
Peut-on préciser `it("Jette une erreur si la demande est déjà traitée pour que le front puisse avertir l'utilisateur")`